apps/nuxt3-ssr/utils/fingerprint.js

```diff
@@ -1 +1 @@
-export const hash = "";
```

Comment:
update ignore te remove gen on build

Reply:
Why haven't you done so yet? I already warned about this construction when this still was a PR...